sdk/eventhub/event-hubs/samples/receiveEventsLoop.ts

```diff
@@ -12,27 +12,40 @@ import { EventHubClient, EventPosition } from "@azure/event-hubs";
 
 // Define connection string and related Event Hubs entity name here
 const connectionString = "";
-const eventHubsName = "";
+const eventHubName = "";
 
 async function main(): Promise<void> {
-  const client = EventHubClient.createFromConnectionString(connectionString, eventHubsName);
+  const client = EventHubClient.createFromConnectionString(connectionString, eventHubName);
   const partitionIds = await client.getPartitionIds();
-  let eventPosition = EventPosition.fromStart();
+  const receiver = client.createReceiver(partitionIds[0], {
+    eventPosition: EventPosition.fromFirstAvailableEvent(),
+    consumerGroup: "$Default"
+  });
   const batchSize = 1;
+  let messageLength = 0;
```

Comment:
Since this gets used only for the iterator, move it to just before using the iterator and rename to suggest that this represents events from the iterator